graphdatascience/error/cypher_warning_handler.py

```diff
@@ -0,0 +1,31 @@
+import warnings
+from functools import wraps
+from typing import Any, Callable, TypeVar, cast
+
+from ..caller_base import CallerBase
+
+F = TypeVar("F", bound=Callable[..., Any])
+
+
+def filter_id_func_deprecation_warning() -> Callable[[F], F]:
```

Comment:
As the id() function is central to GDS, we could also filter out the warning for every query

Reply:
As said in the comment above, I also dont think this is really needed to filter out completly.
Instead, I would consider to log this warning as there is nothing we can do about it.

However, arent we worrying users if they see this warning and there is no way they can fix it?

Reply:
I wonder who the warning is for. If I use GDS Client and `gds.run_cypher("MATCH (n) RETURN id(n)")` I ought to get a warning right?